app/services/decisions/decision_tree.rb

```diff
@@ -64,6 +64,8 @@ class DecisionTree < DslDecisionTree
     from(:case_details).goto(edit: 'nsm/steps/hearing_details')
     from(:hearing_details).goto(edit: 'nsm/steps/case_disposal')
     from(:case_disposal).goto(edit: 'nsm/steps/reason_for_claim')
+    # To be updated when we do ticket: CRM457-2288
```

Comment:
Nit: The syntax we have usually used for these is `# TODO CRM457-2288: [explanation of what needs to change]`